src/tzkt/tzkt_reward_api.py

```diff
@@ -25,14 +25,23 @@ def calc_estimated_reward(self, cycle: int, num_blocks: int, num_endorsements: i
         :param num_blocks: Number of baking rights
         :param num_endorsements: Number of endorsement rights
         """
-        proto = self.api.get_protocol_by_cycle(cycle)
+        # FIXME: https://api.tzkt.io/v1/protocols/cycles/242 always return Granada constants
+        # Temporarly hard-coding the values
 
-        block_reward = \
-            proto['constants']['blockReward'][0] \
-            * proto['constants']['endorsersPerBlock']
+        if cycle >= 388:
```

Comment:
Getting protocol by cycle is fixed now on TzKT. https://api.tzkt.io/v1/protocols/cycles/200

Reply:
Great, code is now restored. https://github.com/tezos-reward-distributor-organization/tezos-reward-distributor/pull/474/commits/e59ecd8acac8f5f3e9950507c65ca29d70bd6028